### ISSUE TITLE
Linter-gcc not working / not even displaying messages in console

### ISSUE THREAD

**Issue by bigboiblue** [AtomLinter/linter-gcc#201]
## Checklist
- [x] Check that you have the GNU C Compiler installed by typing ```gcc --version``` into a terminal.
- [x] If you don't have the ```linter``` package installed, either update to the latest version of ```linter-gcc``` or install linter directly with ```apm install linter```.
- [x] Ensure that you are using a valid grammar - either C/C++/C++14
- [x] Ensure that you have read all the relevant information in the [wiki](https://github.com/hebaishi/linter-gcc/wiki), particularly the [Common Issues](https://github.com/hebaishi/linter-gcc/wiki/Common-Issues) page.

## Issue (including debugging messages):

Hey,
I have installed the latest versions of linter, linter-ui-default and linter-gcc. I am using:
```
Atom    : 1.34.0
Electron: 3.1.1
Chrome  : 66.0.3359.181
Node    : 10.2.0
```
I have also installed the latest gcc-compiler extension to enable me to compile c++ programs within atom. When I compile my C++ programs, which I have purposefully ridden with a bunch of errors, the linter shows no errors. The linter simply does nothing.

I have checked show debugging messages, and there are no messages regarding linter or linter-gcc in the developer console.

I do however repeatedly get this warning in the console:

> `fs.js:958 (electron) The 'lstatSyncNoException' function has been deprecated and marked for removal.`

whenever I perform pretty much any action in atom, however, this may not be related at all, as this warning is displayed regardless of whether or not linter and linter-gcc is installed.

I am on Arch Linux if that bares any relevence.

I would greatly appreciate it if you could help me out with this, as I find this to be a great package.

Thanks

**Comment by bigboiblue** [comment]
I now know the issue: https://github.com/steelbrain/linter/blob/v2.3.0/docs/guides/upgrading-to-standard-linter-v2.md

Would appreciate it if you could update your package to account for this widespread issue.

**Comment by otbrown** [comment]
As it seems linter-gcc is no longer maintained @tvincent056 has published his fork which is compatible with the updated base linter as [linter-gcc2](https://atom.io/packages/linter-gcc2). See [comment](https://github.com/hebaishi/linter-gcc/pull/185#issuecomment-435928445) on #185

**Comment by Arcanemagus** [comment]
Duplicate of #199
